Tasks/Maven/maventask.ts

```diff
@@ -74,6 +74,21 @@ if (specifiedJavaHome) {
     tl.setVariable('JAVA_HOME', specifiedJavaHome);
 }
 
+var ccTool = tl.getInput('codeCoverageTool');
+var isCodeCoverageOpted = (typeof ccTool != "undefined" && ccTool && ccTool.toLowerCase() != 'none');
+
+if (isCodeCoverageOpted) {
+    var summaryFile = null;
+    var reportDirectory = null;
+    var reportPOMFile = null;
+    var execFileJacoco = null;
+    var ccReportTask = null;
+    enableCodeCoverage();
```

Comment:
Can you add type information to the above vars?
`var summaryFile: string = null;`